src/mokernels/intrinsiccoregion.jl

```diff
@@ -0,0 +1,36 @@
+@doc raw"""
+    IntrinsicIntrinsicCoregionMOKernel(kernel::Kernel, B::AbstractMatrix)
+
+Kernel associated with the intrinsic coregionalization model.
+
+# Definition
+
+For inputs ``x, x'`` and output dimensions ``p, p'``, the kernel is defined as[^ARL]
+```math
+k\big((x, p_x), (x', p_{x'}); B, \tilde{k}\big) = [B]_{p_x, p_{x'}} \tilde{k}\big(x, x'\big),
+```
+where ``B`` is a positive semidefinite matrix of size ``m \times m``, with ``m`` being the number of outputs, and ``\tilde{k}`` is a scalar-valued kernel shared by the latent processes.
```

Comment:
92 characters limit:
```suggestion
where ``B`` is a positive semidefinite matrix of size ``m \times m``, with ``m`` being the
number of outputs, and ``\tilde{k}`` is a scalar-valued kernel shared by the latent
processes.
```